### PR TITLE
Aurora MySQL & PostgreSQL: expose instance_class property

### PR DESCRIPTION
**Pull request by zucchinidev**
[#183556008](https://www.pivotaltracker.com/story/show/183556008)

Which other branches should this be merged with (if any)?

Main.

### CODE REVIEW

File: reference/aws-aurora-mysql.html.md.erb [hunk 1]
```diff
@@ -107,14 +107,23 @@ provision only:
...
       <td><code>null</code></td>
       <td>provision and update</td>
     </tr>
     <tr>
       <td><code>serverless_max_capacity</code></td>
       <td>Number</td>
-      <td>The maximum capacity for the cluster. Must be greater than or equal to <code>serverless_min_capacity</code>. Valid capacity values are in a range of 0.5 up to 128, in steps of 0.5.</td>
+      <td>
+        The maximum capacity for the cluster. Must be greater than or equal to <code>serverless_min_capacity</code>. Valid capacity values are in a range of 0.5 up to 128, in steps of 0.5.
+        The `serverless_min_capacity` and `serverless_max_capacity` properties are ineffective without setting the `instance_class` property to a valid value for Aurora Serverless v2.
```
Comment (jhvhs) [inline reference/aws-aurora-mysql.html.md.erb:123]: **suggestion**: I believe the same applies to PostgreSQL as well.

File: reference/aws-aurora-mysql.html.md.erb [hunk 2]
```diff
@@ -107,14 +107,23 @@ provision only:
...
+        Review documentation to understand the restrictions associated with the different types of instances accepted by Aurora:
```
Comment (jhvhs) [inline reference/aws-aurora-mysql.html.md.erb:124]: **thought**: Feel free to ignore this, but I still believe that the `db.serverless` instance type is worth mentioning here. Unlike other instance types, it does not communicate a given configuration, but is rather a placeholder, so it's very unlikely to change in the nearest future. Especially if we're mentioning the exact version of serverless (v2 in this case).

File: reference/aws-aurora-mysql.html.md.erb (hunk 1 above)
Comment (zucchinidev) [reply reference/aws-aurora-mysql.html.md.erb:124]: Thank you!!

File: reference/aws-aurora-mysql.html.md.erb (hunk 2 above)
Comment (zucchinidev) [reply reference/aws-aurora-mysql.html.md.erb:125]: I just added the explanatory note. Thank you very much!! @jhvhs

**Review by jhvhs — APPROVED** [review]

**Pull request merged**